src/util/categories.ts

```diff
@@ -13,6 +13,9 @@ export interface SplitCategory {
  * The subcategory can be localized and freely edited.
  */
 export function splitCategory(fullCategory: string = '', defaultCategory: Category = 'income'): SplitCategory {
+  if (fullCategory.length > 0 && !fullCategory.includes(':')) {
```

Comment:
This is probably the best solution, unless we want to refactor the whole function to use `string.split(':')`.